storey/flow.py

```diff
@@ -194,6 +194,8 @@ def __init__(self, initial_state, fn, group_by_key=False, **kwargs):
         self._state = initial_state
         self._fn = fn
         self._group_by_key = group_by_key
+        if hasattr(initial_state, 'close'):
+            self._closeables = [initial_state]
```

Comment:
Shouldn't this be appended instead?

Reply:
as we discussed, each step sets his `_closeables` list and the Flow.run() is responsible on appending all the closeables into one list

Reply:
Ok, it's just that we already create a list in Flow.__init__ 🤷‍♂️